app/src/renderer/connectors/lcdClient.js

```diff
@@ -95,9 +95,7 @@ Object.assign(Client.prototype, {
   candidates: req("GET", "/stake/validators"),
   getValidators: req("GET", "/validatorsets/latest"),
   queryDelegation: function(delegator, validator) {
-    return req("GET", `/stake/${delegator}/bonding_status/${validator}`).call(
-      this
-    )
+    return req("GET", `/stake/${delegator}/delegation/${validator}`).call(this)
```

Comment:
will this same endpoint keep us from DDoSing ourselves?

Reply:
I don't think so. This was just an API rename.